src/accessibility.md.njk

```diff
@@ -59,21 +59,7 @@ We plan to fix these accessibility issues by the end of June 2021.
 
 ### How this website has been tested for accessibility
 
-The Design System website was last tested on 7 October 2019. The test was carried out by the [Digital Accessibility Centre (DAC)](https://digitalaccessibilitycentre.org/).
-
-DAC tested a sample of pages to cover the different content types in the GOV.UK Design System website. They were:
-
-- [the homepage](https://design-system.service.gov.uk/)
-- [a short content page](https://design-system.service.gov.uk/community/design-system-working-group)
-- [a long content page](https://design-system.service.gov.uk/styles/layout/)
-- [an overview page](https://design-system.service.gov.uk/community/)
-- [a styles page](https://design-system.service.gov.uk/styles/typography/)
-- [a component page](https://design-system.service.gov.uk/components/radios/)
-- [a pattern page](https://design-system.service.gov.uk/patterns/question-pages/)
-
-They also tested the global search functionality that appears in the header of every page in the Design System.
-
-The [GOV.UK Frontend documentation website](http://frontend.design-system.service.gov.uk/) was last tested in July 2020. The test was carried out by the technical writing team at GDS. We used the [WAVE Web Accessibility Evaluation Tool](https://wave.webaim.org/) and a checklist we created with the help of the GDS accessibility team. We tested all the website's pages.
+We last tested this website for accessibility issues in March 2021.
```

Comment:
@36degrees Reason for this addition: the tech writer leading on accessibility fixes said "Because we haven't done all those tests this time around, we thought leaving it as a high-level statement was appropriate."

Reply:
I don't think this change is quite right – we haven't tested the Design System for accessibility issues since 7 October 2019. The most round of testing only apply to the frontend docs.

I'd suggest only changing the section around frontend docs, so something like:

> The Design System website was last tested on 7 October 2019. The test was carried out by the [Digital Accessibility Centre (DAC)](https://digitalaccessibilitycentre.org/).
> 
> DAC tested a sample of pages to cover the different content types in the GOV.UK Design System website. They were:
> 
> - [the homepage](https://design-system.service.gov.uk/)
> - [a short content page](https://design-system.service.gov.uk/community/design-system-working-group)
> - [a long content page](https://design-system.service.gov.uk/styles/layout/)
> - [an overview page](https://design-system.service.gov.uk/community/)
> - [a styles page](https://design-system.service.gov.uk/styles/typography/)
> - [a component page](https://design-system.service.gov.uk/components/radios/)
> - [a pattern page](https://design-system.service.gov.uk/patterns/question-pages/)
>
> They also tested the global search functionality that appears in the header of every page in the Design System.
>
> **The [GOV.UK Frontend documentation website](http://frontend.design-system.service.gov.uk/) was last tested for accessibility issues in March 2021.**

Reply:
That makes sense - thanks. Have changed as per your comment.